Core/clim-basic/extended-streams/text-formatting.lisp

```diff
@@ -44,7 +44,9 @@
                   (setf (getf text-margins ,edge) ,default))))
```

Comment:
Commit message: "stream-text-margin" -> "stream-text-margins"

Reply:
this commit message is correct, it addresses #880 where `:text-margin` - which is the initarg for a slot accessed with `stream-text-margin`, didn't work quite correct with standard-page-layout (and stream-text-margins).